qute.ls/com.redhat.qute.ls/src/main/resources/com/redhat/qute/services/snippets/qute-snippets.json

```diff
@@ -41,6 +41,11 @@
 			"\t$0",
 			"{/if}"
 		],
+		"nestedBody": [
+			"\t$2",
```

Comment:
No, but at first please don't update the qute-snippets.json. Those snippets are for "standard" snippet. Managing nested section means that there is a new one condition : parent section. Indeed {#else} must appears in completion only for `#for` and `#if`.

I think it is doable to manage that with snippet but it need some investigation. Perhaps it will be easier to manage that without snippet.